pom.xml

```diff
@@ -27,7 +27,7 @@
   </parent>
   <groupId>io.fabric8.jenkins.plugins</groupId>
   <artifactId>openshift-sync</artifactId>
-  <version>0.1.31-SNAPSHOT</version>
+  <version>0.1.32-SNAPSHOT</version>
```

Comment:
appreciate the attention to detail, don't update the pom version manually 

I'll deal with that as I cut official versions of the plugin and update the jenkins download center